### PR TITLE
refactor(system): abstract basic CommonStore and migrate scheduler cache

### PR DESCRIPTION
**Pull request by binacs**
First step of https://github.com/kubewharf/godel-scheduler/issues/45

### CODE REVIEW

**Comment by NickrenREN** [comment]
any comments on this ? @yuey002 @lzlaa

File: pkg/testing-helper/framework-helper/wrappers.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```
Comment (yuey002) [inline pkg/testing-helper/framework-helper/wrappers.go:2]: nit, should we update the Copyright header to be godel?

**Review by yuey002 — DISMISSED** [review]

File: pkg/testing-helper/framework-helper/wrappers.go (hunk 1 above)
Comment (binacs) [reply pkg/testing-helper/framework-helper/wrappers.go]: good catch, this should be Godel considering that it is our proprietary file.

Comment (binacs) [reply pkg/testing-helper/framework-helper/wrappers.go]: fixed, PTAL

**Review by NickrenREN — APPROVED** [review]

**Pull request merged**